### PR TITLE
VTA-1680: Add Country to test-stations seed data

### PR DESCRIPTION
**Pull request by m-coslett**
## Add Country to test-stations seed data

In order to determine if an address is in Wales to send a Welsh language certificate, add a field called testStationCountry into the test station seed data. 

[VTA-1680](https://dvsa.atlassian.net/browse/VTA-1680)

## Checklist

- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/functions/putTestStation.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ async function validateTestStation(testStation: any) {
     testStationTown: Joi.string().required(),
     testStationAddress: Joi.string().required(),
     testStationPostcode: Joi.string().required(),
+    testStationCountry: Joi.string().allow(null),
```
Comment (Daniel-Searle) [inline src/functions/putTestStation.ts:33]:
```suggestion
    testStationCountry: Joi.string().required().valid("Wales", "Scotland", "England").allow(null),
```

File: src/models/ITestStation.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export interface ITestStation {
   testStationTown: string;
   testStationAddress: string;
   testStationPostcode: string;
+  testStationCountry?: string;
```
Comment (Daniel-Searle) [inline src/models/ITestStation.ts:12]: Shouldn't this field be required as it's now always going to be coming in from the dynamics feed?
```suggestion
  testStationCountry: string;
```

File: tests/resources/test-stations.json [hunk 3]
```diff
@@ -10,6 +10,7 @@
     "testStationTown": "Paris 20",
     "testStationAddress": "224 Tennyson",
     "testStationPostcode": "36987-2452",
+    "testStationCountry": "USA",
```
Comment (Daniel-Searle) [inline tests/resources/test-stations.json:13]: Why has this been set to USA and not one of the required fields?

File: tests/resources/test-stations.json [hunk 4]
```diff
@@ -241,6 +243,7 @@
     "testStationTown": "Caigangan",
     "testStationAddress": "19461 Esch Drive",
     "testStationPostcode": "59824-0008",
+    "testStationCountry": "",
```
Comment (Daniel-Searle) [inline tests/resources/test-stations.json:246]: Why is this being set as an empty string?

**Review by Daniel-Searle — APPROVED** [review]

**Review by cb-cs — APPROVED** [review]

**Pull request merged**